docs/about/npos-explained.md

```diff
@@ -0,0 +1,73 @@
+networks, ensuring synchronization among nodes. In the absence of consensus in a decentralized blockchain network, there is no guarantee that the state acknowledged by one node
+aligns with the perspectives of other nodes. Consensus strives to establish an objective representation of the state within a network where participants hold subjective views.
+It is the mechanism through which nodes communicate, reach agreement, and collaboratively generate new blocks.
+
+In order to cover Avails concensus we must cover
+
+- **Nominated Proof of Stake (NPOS)** which focuses on who is allowed to take part in concensus
+- **GRANDPA** which focuses on what is the last and final change/block
+- **BABE** which focuses on who will propose the next change/block
+
+Lets dig deeper into Nominated Proof of Stake (NPOS) first.
+
+### Nominated Proof of Stake (NPOS)
+
+Nominated Proof of Stake (NPoS) represents a variation of Delegated Proof of Stake (DPoS), which functions as a consensus algorithm within blockchain networks to establish
+distributed consensus. DPoS entails a set of guidelines for selecting the validator set on blockchains. Instead of all token holders having eligibility to create and validate blocks,
+a specific number of validators, referred to as "delegates" or "witnesses," are chosen to produce blocks and validate transactions. Typically, token holders cast votes to elect these
+validators through a delegation process. In a DPoS system, token holders possess the authority to vote for their preferred delegates, who subsequently take turns producing blocks and
+validating transactions.
+
+The purpose of NPoS aligns with DPoS, aiming to select trustworthy validators for consensus execution. However, the selection mechanism differs. In NPoS, rather than a delegator
+choosing a single validator or specifying which validator their token supports, they nominate a predetermined quantity of validators they are comfortable seeing elected.
+This nomination process gives rise to the term "Nominate Proof of Stake," where the individual staking tokens is referred to as a nominator.
+
+In simple terms, within the Avail ecosystem, you can act as a nominator (or delegator/staker) and nominate up to 16 validators. You do not need to select all 16, you may select any qty under 16.
+
+<img src={useBaseUrl("img/avail/nominator.png")} width="100%" height="100%" />
+
+As a participant with a stake in the network, your contribution may be directed towards endorsing a single validator during a particular era. The illustration below highlights the
+potential scenario where your stake is utilized to back and support a specific validator within the given timeframe
+
+<img src={useBaseUrl("img/avail/nominator-elected-1.png")} width="100%" height="100%" />
+
+Alternatively, your stake may be diversified among multiple validators from the pool of 16 nominees. For instance, in the presented scenario, your stake has been distributed across
```

Comment:
@staking4all hey where do you get the limit of 16 nominable validators from?

Reply:
@Genesis3800 it can be seen on developer->chain state->constants, then select staking and max nominations. 

![image](https://github.com/availproject/availproject.github.io/assets/61656547/129eea8c-a995-4d63-b836-2159a37732e9)


Also if you try staking/nominations can only select 16 validators due to this.